resume-buddy/src/main/java/com/google/sps/data/EnumUtility.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.data;
+
+public class EnumUtility {
+
+  enum Degree {
+    HIGHSCHOOL,
+    ASSOCIATE,
+    BACHELOR,
+    MASTER,
+    DOCTORATE,
+    NOPREF;
+  }
+
+  enum Career {
+    COMPSCI,
+    HEALTHCARE,
+    EDUCATION,
+    ENGINEERING,
+    BUSINESS,
+    OTHER;
+  }
+
+  enum NumYears {
+    LESSTHANFIVE,
+    FIVETOTEN,
+    GREATERTHANTEN,
+    NOPREF;
+  }
+
+  enum SchoolYear {
+    HIGHSCHOOL,
+    FIRSTYEAR,
+    SECONDYEAR,
+    THIRDYEAR,
+    FOURTHYEAR,
+    POSTGRAD;
+  }
```

Comment:
Nit:
```
enum SchoolYear {
  HIGH_SCHOOL,
  YEAR_1,
  YEAR_2,
  YEAR_3,
  YEAR_4,
  POST_GRAD;
}
```

Also, what if someone is in their >= 5th year of undergraduate? This is possible, ie, took more classes or they attend a school like Northeastern University that makes undergraduate engineering programs 5 years to include a co-op program.

Reply:
good point - I added an "Other" option here in case someone does not fit under the given categories